### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aloisbarreras**

### CODE REVIEW

**Comment by aloisbarreras** [comment]
clone it, npm install it, and npm test. It seems to be using the right values when there are integers and floats mixed in together.

**Comment by schnie** [comment]
@aloisbarreras can you remove the dotenv stuff from your test command?

**Comment by aloisbarreras** [comment]
@schnie I couldn't tell what the regex thing was doing in here. Can you help with that? everything else should be good

**Comment by schnie** [comment]
@jlaneve added it to match names with types without adding each field to the schemaHint.  Not sure if it's being used or not.  Not sure if it's the way to go or not.

**Comment by schnie** [comment]
@aloisbarreras What about a mixture of strings and integers?

**Comment by aloisbarreras** [comment]
@schnie added

**Comment by schnie** [comment]
@aloisbarreras can you also add an .eslintrc?

**Comment by aloisbarreras** [comment]
@schnie boom

**Comment by schnie** [comment]
@aloisbarreras I'm wondering if we should keep score of each field type and decide at the end, like the jts-infer package does.  Thoughts?

**Comment by schnie** [comment]
I think the logger module could be removed too.  Not sure we should include that in utility packages like this.  Doesn't seem like its used either.

File: lib/index.js [hunk 1]
```diff
@@ -1,8 +1,93 @@
...
+           return true; 
+        }
+
+        if (originalType === 'integer' && newType === 'decimal') {
+            return true;
+        }
+
+        return false;
+    }
 }
```
Comment (schnie) [inline lib/index.js]: Kill this extra newline?

**Comment by aloisbarreras** [comment]
@schnie I thought that too but scoring wouldn't necessarily work. If there's 99 integers and 1 float, then the integer will win but it'll break when it tries to insert the float

**Comment by schnie** [comment]
@aloisbarreras Could we just make this module a through stream that just reads all chunks, then when the source is ended, just push a single value of the schema, then end?  I'd like it to play nice with the FRP style of writing integrations.

**Comment by aloisbarreras** [comment]
Yes but it works for now, so let's get it working to help Ben out and then I can start looking at that

**Comment by aloisbarreras** [comment]
removed the logger

**Pull request merged**